xmodule/item_bank_block.py

```diff
@@ -497,21 +500,33 @@ def author_view(self, context):
         """
         fragment = Fragment()
         root_xblock = context.get('root_xblock')
-        is_root = root_xblock and root_xblock.usage_key == self.usage_key
-        # User has clicked the "View" link. Show a preview of all possible children:
-        if is_root and self.children:  # pylint: disable=no-member
-            fragment.add_content(self.runtime.service(self, 'mako').render_cms_template(
-                "library-block-author-preview-header.html", {
-                    'max_count': self.max_count if self.max_count >= 0 else len(self.children),
-                    'display_name': self.display_name or self.url_name,
-                }))
+        is_root = root_xblock and root_xblock.location == self.location
```

Comment:
[question] Why `.location` instead of `.usage_key`? I thought `.location` was deprecated?

Reply:
I assume this was just a rebase thing. It was `.location` on my branch (carried over from the original block) until I updated it to `.usage_key` earlier today.

Reply:
Sorry, my bad. Just a rebase issue indeed.

Reply:
Restored :)